client/src/partials/purchases/create/create.html

```diff
@@ -0,0 +1,207 @@
+<div class="flex-header">
+  <div class="bhima-title">
+    <ol class="headercrumb">
+      <li class="static">{{ "TREE.INVENTORY" | translate }}</li>
+      <li class="title">{{ "PURCHASES.PAGE_TITLE" | translate }}</li>
+    </ol>
+  </div>
+</div>
+
+<div class="flex-content">
+  <div class="container-fluid">
+
+    <!--
+      top note panel for the purchase order.details
+      inspired by the patient invoice page's ergonomics
+    -->
+    <div class="row">
+      <div class="col-xs-12">
+        <div class="panel panel-default">
+          <div class="panel-body">
+            <form name="PurchaseOrderForm" autocomplete="off" autocapitalize="off" autocorrect="off" novalidate>
```

Comment:
If this was re-based it could use your [bh-form-defaults](https://github.com/IMA-WorldHealth/bhima-2.X/pull/542)